io_scene_niftools/operators/geometry.py

```diff
@@ -45,31 +45,31 @@
 
 class BsInvMarkerAdd(Operator):
     """Adds BsInvMarker set"""
-    bl_idname = "object.niftools_bs_invmarker_add"
+    bl_idname = "object.bs_inv_marker_add"
```

Comment:
why drop the niftools prefix?

Reply:
To be consistent with the operator's class name and the props & ui.

Reply:
Should we consider a Nif* prefix then for our classes etc. so we can easily identify them ?

Reply:
No need imo.